cmd/minishift/cmd/profile/profile_set.go

```diff
@@ -41,6 +42,10 @@ func runProfile(cmd *cobra.Command, args []string) {
 	validateArgs(args)
 	profileName := args[0]
 
+	if strings.TrimSpace(profileName) == "" {
```

Comment:
-1, there is already a _validateArgs_ function. Any checks should probably happen there. Also please add a unit test.

Reply:
Moved the check to validateArgs function but that function doesn't return anything, just exits with a message, so it means we have to add integration test?? @hferentschik